### DISCUSSION TITLE
change background color after apply background color

### DISCUSSION THREAD

**Discussion by lewbei** [pymupdf/PyMuPDF#1807] [Looking for help] [answered]
I cannot change the background color after apply draw_rect in the pdf. For example, I have apply red by using draw_rect, then I want to change to another color. Is there a way for me to change the rect color or remove the rect? I used redact but it will removed the whole page.

**Comment by JorjMcKie** [comment, marked as answer]
No this is not possible. All draw_* methods of `Page`, resp. `Shape` are "forever".
Redactions can only remove text, images and links.
